yjit/src/codegen.rs

```diff
@@ -2829,6 +2833,7 @@ fn gen_setinstancevariable(
         // If we know the stack value is an immediate, there's no need to
         // generate WB code.
         if !stack_type.is_imm() {
+            asm.spill_temps(); // for ccall
```

Comment:
```suggestion
            asm.spill_temps(); // for ccall (unconditionally spill them for RegTemps consistency)
```

It's not generally required for a regular ccall, so `// for ccall` alone doesn't make sense.